scripts/exec_irmin_bench.py

```diff
@@ -0,0 +1,241 @@
+import os
+import argparse as arg
+import logging as log
+
+monthes = [
+    "01",
+    "02",
+    "03",
+    "04",
+    "05",
+    "06",
+    "07",
+    "08",
+    "09",
+    "10",
+    "11",
+    "12"
+]
+
+years = ["21", "22"]
+
+irmin_versions = ["3.0"]
+
+number_to_mont = {
+    "01": "jan",
+    "02": "feb",
+    "03": "mar",
+    "04": "april",
+    "05": "may",
+    "06": "jun",
+    "07": "july",
+    "08": "aug",
+    "09": "sep",
+    "10": "oct",
+    "11": "nov",
+    "12": "dec",
+}
+
+
+def display_header():
+    print("""
++--------------------------------------------+
+| Name: Bench runner                         |
++--------------------------------------------+
+| Author: Étienne Marais <etienne@maiste.fr> |
+| Version: 20220228                          |
++--------------------------------------------+
+        """)
+
+
+def convert_to_branch(month, year):
+    return "bench-{}-{}".format(number_to_mont[month], year)
+
+
+def verify_type(args):
+    if args.month is not None and args.year is not None:
+        if args.irmin is not None:
+            log.error("Cant have month-year with irmin version")
+            exit(1)
+        else:
+            return 0
+    if args.irmin is not None:
+        if args.month is not None or args.year is not None:
+            log.error("Can't have month-year with irmin version")
+            exit(1)
+        else:
+            return 1
+
+
+def clear_locked():
+    for file in os.listdir():
+        if file.endswith("locked"):
+            log.debug("Remove {}".format(file))
+            os.remove(file)
+
+
+def setup_fake_git():
+    log.info("Setting up a fake git address")
+    os.system("git config --global user.email \"tmp@example.com\"")
+    os.system("git config --global user.name \"Tmp\"")
```

Comment:
It needs correction, it requires a flag to have this! Otherwise, it will break the git system...